README.md

```diff
@@ -6,7 +6,7 @@ This is a development version of Configlet for use with v3 of Exercism.
 
 The application is a single binary and can be used as follows:
 
-```
+```text
```

Comment:
@wolf99 I notice that #462 made this `console` instead - is that better? 

>  - `console` is an alias for a shell *session*
> (https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L5689) - i.e. a block that includes the prompt and output.

seems not apply to this block - for that we'd need to add `$ configlet --help` at the top, right?

Reply:
Could reasonably be either I suppose.
I changed to `console` based on this being the output we see in a session when `configlet --help` is used, despite the command itself not being included.
If you think that does't make sense I think it's OK to set it to `text`, or could include the command and leave at `console` ?

Reply:
Thanks.

Let's go with `text` for now, and make it `console` in the future if we add `$ configlet --help` at the start.